src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -125,11 +113,6 @@ public static function register($handler = null, $replace = true)
         $levels = -1;
 
         if ($handlerIsNew = !$handler instanceof self) {
```

Comment:
this condition could become ``null === $handler`` rather than doing an instanceof check